aws/cloudformation/data.yml.erb

```diff
@@ -97,27 +97,17 @@ Resources:
             Resource:
               - 'arn:aws:s3:::cdo-data-sharing-internal/*'
               - 'arn:aws:s3:::cdo-data-sharing-internal'
-  RDSEndpoint:
+  ProductionMySQLDatabaseReportingEndpoint:
```

Comment:
Noticing the word `Production` here. What does this word mean in this context? Especially given the use of `AWS::StackName` inside of it.

Reply:
renamed to `MySQLDatabaseSourceEndpoint`